### DISCUSSION TITLE
My contributions are not displayed on the graph

### DISCUSSION THREAD

**Discussion by ln-dev7** [community/community#32850] [Other Feature Feedback, Questions, & Ideas] [answered]
For several weeks there are some commits that I do but they are not displayed on the graph and I don't know the cause of the problem because I didn't modify anything in my settings.
I just made a commit on a repo, when I enter the network I see the commit but not the contribution on the graph

**Comment by radar-blip** [comment]
Did you try refreshing the page?
Also, if its a private repo, I dont think commits will show up there.

**Comment by RedoxOwn** [comment, marked as answer]
Hello my friend, There are many reasons why your contributions are not showing up on your profile.
Because contributions are only counted if they meet certain criteria so not **all** contributions will show in your profile.

First:
Issues, pull requests and discussions will only appear on your contribution graph if they were opened in a standalone repository, **not** a fork.

Second:
Commits will appear on your contributions graph if they meet all of the following conditions:
- The email address used for the commits is associated with your account on GitHub.com.
- The commits were made in a standalone repository, not a fork.
- The commits were made:
  1. In the repository's default branch
  2. In the gh-pages branch (for repositories with project sites)

Third:
In addition, at least one of the following must be true:
- You are a collaborator on the repository or are a member of the organization that owns the repository.
- You have forked the repository.
- You have opened a pull request or issue in the repository.
- You have starred the repository.

I hope this clarifies things for you 😸

**Reply by ln-dev7** [reply]
Thanks so much !

**Reply by RedoxOwn** [reply]
Your welcome brother 😸
By the way, the status of your question is still **Unanswered**, you can mark my answer as the correct one to your question so that other developers can find it in the future

**Reply by zaidihusain** [reply]
Hi, I'm facing an issue where my GitHub contribution graph is not updating (green squares are not appearing), even though:

The email in my local Git configuration matches my GitHub account email.
I’m pushing to a public repository that I own.
My commits are being pushed to the default branch (main).
I’m adding new files or modifying existing files, so there are valid changes.
The Boxes go Green only when i push a new repo but it doesnt go green when i push changes in the same repo
